### PR TITLE
Make movement descriptions display linebreaks

### PR DESCRIPTION
**Pull request by Yori-O92**

### CODE REVIEW

File: gridt/src/app/movements/movements-detail/movements-detail.page.html [hunk 1]
```diff
@@ -13,9 +13,9 @@
       <ion-row>
         <ion-col size-md="6" offset-md="3">
           <h2>{{ (movement$ | async)?.name }}</h2>
-          <p><strong>{{ (movement$ | async)?.short_description }}</strong></p>
-          <p *ngIf="(movement$ | async )?.description">{{ (movement$ | async)?.description }}</p>
-
+          <p class="shortdescription">{{ (movement$ | async)?.short_description }}</p>
```
Comment (Drvanon) [inline gridt/src/app/movements/movements-detail/movements-detail.page.html:16]: Why did you remove the strong?

Comment (Yori-O92) [reply gridt/src/app/movements/movements-detail/movements-detail.page.html:16]: The font type is now included in the css class for the shortdescription as well. It comes out the same.

Comment (Drvanon) [reply gridt/src/app/movements/movements-detail/movements-detail.page.html:16]: That is a code design decision. I don't know how much you know about this, but if you are unfamiliar with semantic html, I would recommend reading the following article. I don't necessarily argue one way or the other, but I do think that it is useful to make it in an informed manner.

https://medium.com/adalab/the-importance-of-semantic-html-78e74fb75ff0

Comment (Yori-O92) [reply gridt/src/app/movements/movements-detail/movements-detail.page.html:16]: Interesting consideration and good to become aware of. However, I think that in this case, the <strong> tag has a layout function than rather than a semantic one. Therefore, I would still leave it in the scss file.

Comment (Drvanon) [reply gridt/src/app/movements/movements-detail/movements-detail.page.html:16]: Okay, then I'll approve your pull request.
